src/ft2font_wrapper.cpp

```diff
@@ -575,19 +584,20 @@ static PyObject *PyFT2Font_set_text(PyFT2Font *self, PyObject *args, PyObject *k
 
 const char *PyFT2Font_get_num_glyphs__doc__ =
     "get_num_glyphs()\n"
-    "\n"
-    "Return the number of loaded glyphs\n";
+    "--\n\n"
+    "Return the number of loaded glyphs.\n";
 
 static PyObject *PyFT2Font_get_num_glyphs(PyFT2Font *self, PyObject *args, PyObject *kwds)
 {
     return PyLong_FromLong(self->x->get_num_glyphs());
 }
 
 const char *PyFT2Font_load_char__doc__ =
-    "load_char(charcode, flags=LOAD_FORCE_AUTOHINT)\n"
-    "\n"
-    "Load character with charcode in current fontfile and set glyph.\n"
-    "The flags argument can be a bitwise-or of the LOAD_XXX constants.\n"
+    "load_char(charcode, flags=32)\n"
```

Comment:
Why is this one 32 now, when `load_text` doesn't show a default in the signature at all?

Reply:
because I had missed that